Exec/science/wdmerger/wdmerger_util.cpp

```diff
@@ -0,0 +1,71 @@
+#include <wdmerger_util.H>
+
+extern "C" {
+
+// Given total mass of a binary system and the initial separation of
+// two point particles, obtain the velocity at this separation
+// assuming the point masses fell in from infinity. This will
+// be the velocity in the frame where the center of mass is stationary.
+
+void freefall_velocity (Real mass, Real distance, Real& vel)
+{
+    vel = std::sqrt(2.0_rt * C::Gconst * mass / distance);
+}
+
+// Accepts the masses of two stars (in solar masses)
+// and the orbital period of a system,
+// and returns the semimajor axis of the orbit (in cm),
+// as well as the distances a_1 and a_2 from the center of mass.
+
+void kepler_third_law (Real radius_1, Real mass_1, Real radius_2, Real mass_2,
+                       Real& period, Real eccentricity, Real phi, Real& a,
+                       Real& r_1, Real& r_2, Real& v_1r, Real& v_2r, Real& v_1p, Real& v_2p)
+{
+    Real length;
+
+    Real M  = mass_1 + mass_2;     // Total mass
+    Real mu = mass_1 * mass_2 / M; // Reduced mass
+
+    // First, solve for the orbit in the reduced one-body problem, where
+    // an object of mass mu orbits an object with mass M located at r = 0.
+    // For this we follow Carroll and Ostlie, Chapter 2, but many texts discuss this.
+    // Note that we use the convention that phi measures angle from aphelion,
+    // which is opposite to the convention they use.
+
+    if (period > 0.0_rt && a < 0.0_rt)
+    {
+        a = std::pow(C::Gconst * M * period * period / (4.0_rt * M_PI * M_PI), 3); // C + O, Equation 2.37
+    }
```

Comment:
shouldn't this be to the 1/3 power? not 3?

Reply:
Good catch, fixed